paymentsheet/src/test/java/com/stripe/android/customersheet/state/DefaultCustomerSheetLoaderTest.kt

```diff
@@ -53,6 +57,9 @@ class DefaultCustomerSheetLoaderTest {
     private val readyGooglePayRepository = mock<GooglePayRepository>()
     private val unreadyGooglePayRepository = mock<GooglePayRepository>()
 
+    @get:Rule
+    val featureFlagTestRule = FeatureFlagTestRule(FeatureFlags.customerSheetACHv2, isEnabled = true)
```

Comment:
Should this use `isEnabled = false` to reflect production behavior?

Reply:
Sure, I can flipflop.